docs/admin/provisioners.md

```diff
@@ -47,13 +47,12 @@ the [Helm example](#example-running-an-external-provisioner-with-helm) below.
 
 ## Types of provisioners
 
-> Provisioners have two important tags: `scope` and `owner`. Coder sets these
-> tags automatically.
+Provisioners can broadly be categorized by scope: `organization` or `user`.
```

Comment:
It might be helpful to add some text about how this difference comes about, e.g.

External provisioners started using a pre-shared key, or an admin API key (deprecated) are organization-scoped.  Provisioners started in-process with Coder server are organization-scoped.

External provisioners started using a non-admin API key are user-scoped to the corresponding user.